### DISCUSSION TITLE
iFrame becomes pixelated when model is moving

### DISCUSSION THREAD

**Discussion by zacksuper** [google/model-viewer#3782] [Q&A]
Hi everyone,

I noticed in the last couple of months that anytime the model is moving, it becomes fuzzy/pixelated. I noticed this on all of the modelviewer examples as well, so I know it's not specific to the way I set it up. Is there a way to force the hi-res version even while the model is moving? When it is stationary, the full-res version loads, but the second the user interacts or an animation plays, it becomes blurry until it stops moving again.

This is happening for me on chrome both desktop and mobile versions.

Any help would be greatly appreciated! 

Thanks,
Zack

**** **UPDATE** ****
After a bit more digging I found the Dynamic Scaling portion of the documentation! Disregard this discussion, I think the solution lies there.

**Comment by elalish** [comment]
Indeed, this is a feature, not a bug, and it sounds like you've found the docs to control it. If you find a situation where limiting the render scaling *does not* result in a lower frame-rate, please post an example for us here. That would imply it's not working as intended.

**Reply by zacksuper** [reply]
Awesome! Thanks a lot for the reply elalish. Modelviewer has completely changed the way I think about including 3d on the web and removed so many insane barriers to entry for me coming from a non-coding background.

And for anyone who finds this post down the line, here's the [anchor link](https://modelviewer.dev/examples/loading/#renderScale) to the documentation i'm referring to above!
